x/auth/ante/sigverify_test.go

```diff
@@ -44,7 +44,7 @@ func TestSetPubKey(t *testing.T) {
 	// set accounts and create msg for each address
 	for i, addr := range addrs {
 		acc := suite.accountKeeper.NewAccountWithAddress(suite.ctx, addr)
-		require.NoError(t, acc.SetAccountNumber(uint64(i)))
+		require.NoError(t, acc.SetAccountNumber(uint64(i+1000)))
```

Comment:
should we open an issue to redo these tests to avoid this overwriting?

Reply:
TBH, considering we're creating a new auth & accounts module I think we can leave things as is in x/auth assuming no more huge extra features will be put here.